src/Components/Web.Extensions/src/InputFile/Block.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    internal readonly struct Block
```

Comment:
Minor point, but I'd recommend naming or namespacing this to clarify it's related to file uploads. Not a big deal since it's `internal` but might help with ambiguity in the future!